test/labeling/model/test_label_model.py

```diff
@@ -59,6 +59,53 @@ def test_mv_default(self):
             label_model.predict(L), np.array([1, 1, 1])
         )
 
+    def test_prec_init(self):
+        label_model = LabelModel(cardinality=2, verbose=False)
+        L = np.array([[-1, -1, 1], [-1, 1, -1], [0, -1, -1]])
+
+        # test without prec_init
+        label_model.fit(L_train=L, n_epochs=1000, seed=123)
+        Y_pred = label_model.predict(L)
+        assert set(Y_pred.flatten()) == {0, 1}
```

Comment:
What are we checking specifically with the values of Y_pred?

Reply:
Just making sure predicting gives either a 0 or a 1 without an error.

Reply:
Got it. It seems like the fact that the outputs are all either 0 or 1 is not the focus of this test, so we can just run `label_model.predict(L)` to make sure it completes without error